README.md

```diff
@@ -58,6 +58,9 @@ default to `docker`.
 cargo install cross --git https://github.com/cross-rs/cross
 ```
 
+It's also possible to directly download the pre-compiled [release
+binaries](https://github.com/cross-rs/cross/releases).
+
```

Comment:
can we have a little mention to https://github.com/cargo-bins/cargo-binstall here maybe?? what do you think?

Reply:
Yes why not, let's mention it as well :)